### PR TITLE
Add build instructions

### PR DESCRIPTION
**Pull request by brian-rose**
As discussed in #9, this PR adds basic instructions to the README and the Contributor page for how to render the site locally.

### CODE REVIEW

**Review by clyne — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
+This is the source repository for the [Project Pythia portal](https://projectpythia.github.io).
+The portal site is built with [sphinx](https://www.sphinx-doc.org/).
+
+To build the site locally (e.g. for testing new content),
+use [conda](https://docs.conda.io/) to set up a build environment with all dependencies.
+
+First, make a local clone of this source repository on your machine.
+Then, from the `content` directory of the source repository, do this:
+```
+conda env create -f ci/environment.yml
```
Comment (kmpaul) [inline README.md:14]: Should this be...

```suggestion
conda env create -f ../ci/environment.yml
```

**Review by kmpaul — APPROVED** [review]
I approve this. There are a few suggestions, but I can't test them right now so I actually don't know if they are correct. If so, just accept the suggestions.

File: README.md (hunk 1 above)
Comment (brian-rose) [reply README.md:14]: You are correct on both counts! Thanks for this fix.

**Comment by brian-rose** [comment]
Thanks @kmpaul, good catch. 

Is this ready to merge?

**Pull request merged**